mne/preprocessing/ica.py

```diff
@@ -214,7 +214,7 @@ def __init__(self, n_components=None, max_pca_components=None,
         self.max_pca_components = max_pca_components
         self.n_pca_components = n_pca_components
         self.ch_names = None
-        self.random_state = random_state if random_state is not None else 42
+        self.random_state = random_state
```

Comment:
Either we fixed the seed or we don't. Currently seed was used in PCA but not in infomax